CONTRIBUTING.md

```diff
@@ -1,14 +1,110 @@
+>[!NOTE]
+> This is not the repo for Express JS. To contribute to the _[Express JS framework](https://github.com/expressjs/express)_ itself, [check this out](#were-the-express-js-frameworks-documenntation-team---not-the-express-js-framework).
 
-Content on this site is licensed under a Creative Commons Attribution-ShareAlike 3.0 United States License.  See https://creativecommons.org/licenses/by-sa/3.0/us/ for a layman's summary; 
-See [LICENSE.md](LICENSE.md) for the full license.
+If you want to contribute to [https://github.com/expressjs/expressjs.com](https://github.com/expressjs/expressjs.com), you're in the right place. 
 
+
+Whether you're a first time first time contributor or you're just looking for a refresher, visit our [Getting Started](#contributors-guide-to-getting-started) guide below to get up and running.
+
+#### Want to help but need some ideas? These are some of our most typical issue types:
+
+1.  **Website Related**: 
+If you see anything on the site that could use a tune-up, think about how to fix it.
+
+    - display or screen sizing problems
+    - mobile responsiveness issues
+    - missing or broken accessibility features 
+    - website outages
+    - broken links
+    - page structure or user interface enhancements
+
+
+2. **Content Related**: spelling errors, incorrect/outdated Express documentation, adding missing content.
+    - _We love any reports of typos. If you see one, fixing it is a great way to help_.
+
+
+3. **Translation Related**: spelling errors, incorrect/poorly translated words, adding new full site translations.
+> [!IMPORTANT]
+> All translation submissions are currently paused. See this [notice](#notice-we-have-paused-all-translation-contributions) for more information.
+
+   - _If you want to do a full site translation, or just translate a single page, there's a section specifically related to [translations](#contributing-translations) below_.
+
+#### Want to work on a backlog issue?
+
+We often have bugs or enhancements that need work. You can find these on our repo [https://github.com/expressjs/expressjs.com/issues](https://github.com/expressjs/expressjs.com/issues). Check out the tags to find something that's a good match for you.
+
+#### Have an idea? Found a bug?
+
+If you've found a bug or a typo, or if you have an idea for an enhancement, please submit a [new issue](https://github.com/expressjs/expressjs.com/issues/new?assignees=&labels=&projects=&template=3other.md). You can do this by going to our [repo](https://github.com/expressjs/expressjs.com) and opening a "New Issue" under the **Issues** tab. If you submit it, we will respond.
+
+## Contributor's Guide to Getting Started
+
+If you want to learn about working on Expressjs.com, this is the right place. Follow the steps below to get started.
+
+#### TL;DR
+1. Open an issue and get approval.
```

Comment:
is approval necessary?  who is delegated the authority to make these approvals?

Reply:
Here is another instance where you used this. https://github.com/expressjs/expressjs.com/pull/1568#issuecomment-2325137955 which resulted in https://github.com/expressjs/expressjs.com/issues/1590

Reply:
I've made some changes to the wording in those sections and removed the TL;DR part. 
I tried to soften it, make it seem optional, remove all references to "approval", and and say it's for large changes only.